demos/smart_classroom_demo/src/action_detector.cpp

```diff
@@ -108,6 +107,11 @@ ActionDetection::ActionDetection(const ActionDetectorConfig& config)
                 const auto anchor_dims = outputInfo[glob_anchor_name]->getDims();
                 anchor_height = new_network_ ? anchor_dims[2] : anchor_dims[1];
                 anchor_width = new_network_ ? anchor_dims[3] : anchor_dims[2];
+                decltype(anchor_dims.size()) action_dimention_idx = new_network_ ? 1 : 3;
```

Comment:
Using `decltype` for this is a bit silly. You know it's going to be `size_t`.

Reply:
replaced

Reply:
replaced